tools/emcoverage.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import errno
+import os
+import shutil
+import sys
+import uuid
+from glob import glob
+
+from coverage.cmdline import main
+
+if __name__ == '__main__':
+  sys.executable = os.path.abspath(__file__)
+
+  store = os.path.join(os.path.dirname(sys.executable), 'coverage')
+  normal_run = True
+
+  if len(sys.argv) < 2 or sys.argv[1] == 'help':
+    print('Emscripten coverage helper')
+    print('Run emscripten commands like: ./tools/emcoverage.py emcc ...')
+    print('Run ./tools/emcoverage.py report/html/xml to get reports')
+    print('Run ./tools/emcoverage.py reset to clean up old data and start anew')
```

Comment:
Can you expand this documentation? I wasn't clear on what to do from reading just this.